### PR TITLE
Global destination Config

### PR DESCRIPTION
**Pull request by kcoleman731**
* Added feature to allow for global configuration of destination for multiple subtasks.

### CODE REVIEW

File: lib/xctasks/test_task.rb [hunk 1]
```diff
@@ -238,7 +240,7 @@ def namespaced?
       def run_tests(options = {})
         ios_version = options[:ios_version]
         XCTasks::Command.run(%q{killall "iPhone Simulator"}, false) if sdk == :iphonesimulator
-
+
```
Comment (blakewatters) [inline lib/xctasks/test_task.rb]: What’s with this whitespace?

File: lib/xctasks/test_task.rb [hunk 2]
```diff
@@ -297,22 +299,24 @@ def write_environment_variables_to_scheme
...
     def initialize(namespace_name = :test)
       @namespace_name = namespace_name
       @config = Configuration.new
       @subtasks = []
+      @destination
       @namespace_name = namespace_name.kind_of?(Hash) ? namespace_name.keys.first : namespace_name
       @prepare_dependency = namespace_name.kind_of?(Hash) ? namespace_name.values.first : nil
 
       yield self if block_given?
+
```
Comment (blakewatters) [inline lib/xctasks/test_task.rb]: What’s with this whitespace?

File: lib/xctasks/test_task.rb [hunk 3]
```diff
@@ -297,22 +299,24 @@ def write_environment_variables_to_scheme
...
       raise ConfigurationError, "A workspace or project must be configured" unless workspace || project
       raise ConfigurationError, "At least one subtask must be configured" if subtasks.empty?
       define_rake_tasks
     end
-
+
```
Comment (blakewatters) [inline lib/xctasks/test_task.rb]: What’s with this whitespace?

File: spec/test_task_spec.rb [hunk 4]
```diff
@@ -350,6 +350,7 @@
         end
       end
     end
+
```
Comment (blakewatters) [inline spec/test_task_spec.rb]: What’s with this whitespace?

File: lib/xctasks/test_task.rb (hunk 3 above)
Comment (kcoleman731) [inline lib/xctasks/test_task.rb]: this one seems like you want to keep right? Space between an `end` and `def`?

**Pull request merged**